deploy/apache/sites-available/privacyidea.conf

```diff
@@ -11,6 +11,14 @@
 		AllowOverride None
 	</Directory>
 
+        # Yubico servers use /wsapi/2.0/verify as the path in the
+        # validation URL. Some tools (e.g. Kolab 2fa) let the 
+        # user/admin change the api host, but not the rest of
+        # the URL. Let's redirect the api URL to privacyideas
+        # /ttype/yubikey.
```

Comment:
Please put this in comments.
Most setups do not use the yubikey in the yubico cloud mode. And there is no need to turn on the RewirteEngine. 

So installations that use the yubikey cloud mode can simple uncomment these two lines. Cool.